bin/compile

```diff
@@ -29,13 +29,13 @@ if [ -f $ENV_DIR/CHROMEDRIVER_VERSION ]; then
   VERSION=$(cat $ENV_DIR/CHROMEDRIVER_VERSION)
 else
   topic "Looking up latest chromedriver version"
-  LATEST="https://chromedriver.storage.googleapis.com/LATEST_RELEASE"
-  VERSION=`curl -s $LATEST`
+  LATEST="https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json"
+  VERSION=`curl -s $LATEST | grep -Eo '"channel":"Stable","version"[^,]*' | grep -Eo '[^:]*$' | tr -d '"'`
```

Comment:
`jq` is installed in the build images, so we can use that to parse the file :-)

I've made a few tweaks to the existing curl command to make it align with the best practices we use in our official language buildpacks:
- The `--show-error` (aka `-S`) ensures that errors are shown if any occur, since otherwise silent mode (`-s` / `--silent`) hides everything (which makes failures really confusing in the build log)
- The `--fail` (aka `-f`) ensures that non-HTTP-200 response codes actually cause curl to fail, otherwise it will happily pass eg HTTP 4xx/5xx response bodies into the grep/jq, giving confusing errors
- The connect timeout helps prevent hanging builds
- The retries help improve reliability
- I've used the long flag names to make the options passed easier to understand

I've also switched the (existing) deprecated bash backtick usages to `$()`.

```suggestion
  VERSION=$(curl --silent --show-error --fail --retry 3 --retry-connrefused --connect-timeout 10 "${LATEST}" | jq -r '.channels.Stable.version')
```

Reply:
That makes sense. Thanks for taking a look at this!